src/library/entities/Book.java

```diff
@@ -5,91 +5,91 @@
 @SuppressWarnings("serial")
 public class Book implements Serializable {
 	
-	private String tItLe;
-	private String AuThOr;
-	private String CALLNO;
-	private int iD;
+	private String title; //Changed 'tItLe' to 'title' 
+	private String author;//Changed 'AuThOr' to 'author'
+	private String callNo;//Changed 'CALLNO' to 'callNo'
+	private int id;//Changed 'iD' to 'id'
 	
-	private enum sTaTe { AVAILABLE, ON_LOAN, DAMAGED, RESERVED };
-	private sTaTe StAtE;
+	private enum State { AVAILABLE, ON_LOAN, DAMAGED, RESERVED };//Changed 'sTaTe' to 'State'
+	private State state;//Changed ' sTaTe StAtE' to ' State state'
 	
 	
 	public Book(String author, String title, String callNo, int id) {
-		this.AuThOr = author;
-		this.tItLe = title;
-		this.CALLNO = callNo;
-		this.iD = id;
-		this.StAtE = sTaTe.AVAILABLE;
+		this.author = author;// Changed 'AuThOr' to 'author'
+		this.title = title;// Changed 'tItLe ' to 'title'
+		this.callNo = callNo;//Changed 'CALLNO' to 'callNo'
+		this.id = id;//Changed 'iD' to 'id'
+		this.state = state.AVAILABLE;//Changed 'StAtE ' to 'state' and 'sTaTe.AVAILABLE' to 'state.AVAILABLE'
 	}
 	
 	public String toString() {
 		StringBuilder sb = new StringBuilder();

```

Comment:
As the variable name should be meaning full, please use stringBuilder instead of sb.